lxd/apparmor/instance_qemu.go

```diff
@@ -74,8 +74,9 @@ profile "{{ .name }}" flags=(attach_disconnected,mediate_deleted) {
   /snap/lxd/*/share/qemu/**                 kr,
 
   # Snap-specific paths
-  /var/snap/lxd/common/ceph/**              r,
-  {{ .rootPath }}/etc/ceph/**               r,
+  /var/snap/lxd/common/ceph/**                         r,
+  {{ .rootPath }}/etc/ceph/**                          r,
+  {{ .rootPath }}/run/systemd/resolve/stub-resolv.conf r,
```

Comment:
I'd probably add `/run/systemd/resolve/resolv.conf` in there for those not using `systemd-resolved` in stub mode.